src/libs/actions/Policy/Policy.ts

```diff
@@ -3792,6 +3792,27 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
             },
         ],
     };
+    const policyTagList = allPolicyTags?.[policyID];
+    if (!policyTagList) {
```

Comment:
do we need an `isEmpty` check here?

Reply:
I think we don't need `policyTagList` will never be an empty object.